src/api/java/baritone/api/Settings.java

```diff
@@ -1118,6 +1118,11 @@
      */
     public final Setting<Boolean> renderSelectionCorners = new Setting<>(true);
 
+    /**
+     * Use sword to mine.
+     */
+    public final Setting<Boolean> useSwordToMine = new Setting<>(false);
```

Comment:
wait a sec - the current existing behavior is to allow swords, right? because when useSwordToMine is false, you continue in the toolset loop. the current behavior would be `useSwordToMine true`, right? so shouldn't this default to true?

Reply:
Further up in this thread people stood that it should default to false, why would you want to use your sword?

Reply:
Cobwebs